static/app/components/charts/utils.tsx

```diff
@@ -102,12 +104,8 @@ export class GranularityLadder {
   getInterval(minutes: number): string {
     if (minutes < 0) {
       // Sometimes this happens, in unknown circumstances. See the `getIntervalForMetricFunction` function span in Sentry for more info, the reason might appear there. For now, a reasonable fallback in these rare cases is to return the finest granularity, since it'll either fulfill the request or time out.
-      Sentry.withScope(scope => {
-        scope.setFingerprint(['invalid-duration-for-interval']);
-        Sentry.captureException(
-          new Error('Invalid duration supplied to interval function')
-        );
-      });
+      // eslint-disable-next-line @typescript-eslint/no-unused-expressions
```

Comment:
@AbhiPrasad I'm getting eslint complaints if I try to use the literal, can you check my branch to see if it's just me?